perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -15,6 +15,19 @@ sub is_council {
     1;
 }
 
+sub has_feature {
+    my ($self, $feature) = @_;
+    my $features = FixMyStreet->config('FEATURES');
```

Comment:
It'd be good to document this in `general.yml-example`.

Reply:
Yeah, I wondered about that, if it's UK cobrand only, it might be confusing, but I guess there are other things like tat in there already. Perhaps rename it UKFEATURES or something...

Reply:
Or promote has_feature up to Default.pm?

Reply:
Good idea, have done that!